Formula/node_exporter.rb

```diff
@@ -0,0 +1,64 @@
+class NodeExporter < Formula
+  desc "Prometheus exporter for machine metrics"
+  homepage "https://prometheus.io/"
+  url "https://github.com/prometheus/node_exporter/archive/v0.15.2.tar.gz"
+  sha256 "940b850376f94580e88b5e99926d92899975d5792d05709cbd9c48dab0a848ad"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/prometheus"
+    ln_sf buildpath, buildpath/"src/github.com/prometheus/node_exporter"
+
+    system "make", "build"
+    bin.install "node_exporter"
+  end
+
+  plist_options :startup => true
```

Comment:
`:startup => true` should not be used unless this software must be run as root. Is that the case?

Reply:
Documentation doesn't mention root, only loading at startup. Being a monitoring agent, I assume users definitely want it running after startup.

http://www.rubydoc.info/github/Homebrew/brew/Formula.plist_options

Reply:
My bad. I was 100% certain I had put in username/group = nobody (the Prometheus-people are very insisting that it should run with the least possible privileges), but I can see that must have slipped in some rebase.

I tested a bit and eventually found https://github.com/Homebrew/homebrew-services/issues/70 and implemented setting username through `sudo`, as suggested there (and now it starts as `nobody`...)

Let me know if that's the right way to go.

Reply:
Only downside is that `brew services start node_exporter` silently breaks. It only works with `sudo`...